src/Faithlife.Utility/StringUtility.cs

```diff
@@ -890,6 +890,31 @@ public static string Reverse(this string value)
 		public static TextReader CreateDecompressingTextReader(Stream stream, Ownership ownership) =>
 			new DecompressingTextReader(stream, ownership);
 
+		public static bool ContainsOrdinal(this string source, char value) =>
+#if NETSTANDARD2_0
+			source.Contains(value);
+#else
+			source.Contains(value, StringComparison.Ordinal);
+#endif
+
+		public static bool ContainsOrdinal(this string source, string value) =>
+#if NETSTANDARD2_0
+			source.Contains(value);
+#else
+			source.Contains(value, StringComparison.Ordinal);
+#endif
+
+		public static bool ContainsAllOrdinal(this string source, IEnumerable<string> values) =>
+			values.All(source.ContainsOrdinal);
+
+		public static bool ContainsAllOrdinal(this string source, params string[] values) =>
+			source.ContainsAllOrdinal(values.AsEnumerable());
+
+#if !NETSTANDARD2_0
+		public static bool ContainsAll(this string source, IEnumerable<string> values, StringComparison comparisonType) =>
+			values.All(value => source.Contains(value, comparisonType));
+#endif
```

Comment:
Didn't think this one would be useful enough in .NET Standard 2.0 to bother adding our own implementation of `String.Contains(String, StringComparison)`.